README.md

```diff
@@ -0,0 +1,17 @@
+# Pony
+
+Because everybody wants a pony.
+
+## Playing
+
+	. ./bin/activate-hermit
+	make install
```

Comment:
nit: i don't think we should be installing programs called "client" and "server". These instructions would work just as well with `make build`, since the hermit env is activated and out/bin will be in the path.

This suggests that the Makefile should change - adding the concept of "external" and "internal" binaries. Only "external" binaries are installed with `make install`.

Something to think about. Can be addressed later.

Reply:
`make install` with activated hermit env installs to `out/bin` - to me this is local enough and less characters are easier to parse.

here too i will to the wrong thing to make you feel right :lollipop: :cookie:

Reply:
ok, so as you said, GOBIN is $O/out/bin, so no big deal. ignore this comment.

Reply:
As discussed on Slack - `make install` installs to local GOBIN in out/bin. Will leave for now.